src/app.ts

```diff
@@ -1,8 +1,16 @@
 import express from "express";
+import session from "express-session";
 import nunjucks from "nunjucks";
 import bodyParser from "body-parser";
+import { config } from "dotenv";
 
 import { getOpenJobRoles } from "./controllers/JobRoleController";
+import { getLoginForm, postLoginForm, postLogout } from "./controllers/AuthController";
+
+config();
+if (!process.env.SESSION_KEY) {
+    throw Error("Please specify a SESSION_STORAGE environment variable");
+}
```

Comment:
this doesn't add up - like the JWT_KEY make sure you document how to work with this locally and think about it in prod